src/clarity-angular/modal/modal.ts

```diff
@@ -26,7 +27,7 @@ import {GHOST_PAGE_ANIMATION} from "./utils/ghost-page-animations";
     viewProviders: [ScrollingService],
     templateUrl: "./modal.html",
     styles: [`
-        :host { display: inline-block; }
+        :host { display: inline; }
```

Comment:
Why was this changed to `inline` from `inline-block`? I thought we just had to add `display: none` when the modal isn't open. I think thats what @mathisscott mentioned here: https://github.com/vmware/clarity/issues/1622#issuecomment-339349623

Reply:
@adityarb88  because when Popup is open. Due to `inline-block` margin can bee seen in background.

Like When you open modal  in backdrop you can see margin again. So I changed `inline-block ` to `inline`